docs/user_guide/provisioning_tool.rst

```diff
@@ -354,39 +363,22 @@ will ask you if you would like to have one sample copy of this file created.
+Edit the project.yml configuration file to meet your project requirements:
 
-    - "api_version" must be 2 for current release of provisioning tool
-    - "he_config" describes the Homomorphic Encryption configuration
-    - "config_folder" is the folder name that contains configuration information
-    - "disable_authz" is a boolean to instruct FL server whether to enforce role-based rights on admin users. true means all admin users are in role super.
+    - "api_version" must be 3 for current release of provisioning tool
     - "name" is used to identify this project.
-    - "server" section describes the FL server.
-        - "org" is for the name of the owner of this server.
-        - "cn" is the "Fully Qualified Domain Name" and it is very important that this is correct. If this information is not completely correct, the security handshake between the server and clients will fail. Please note that this cannot just be an IP address.
-        - "fed_learn_port" is the port number for communication between the FL server and FL clients
-        - "admin_port" is the port number for communication between the FL server and FL administration client
-        - "admin_storage" is directory name, related to the WORKSPACE, to store files by admin process on server
-        - "min_num_clients" is the minimum number of clients for federated learning to begin
-        - "max_num_clients" is the maximum number of clients allowed in this instance of federated learning
-        - "config_validator" is the class that validates if the uploaded app has correct syntax
-    - The "fl_clients" section describes the FL clients, with one "org", "client_name", and "email" for each client. Please note that each "client_name" must be unique. It will show in the admin console.
-    - The "admin_clients" section describes the FL admin clients. The "email" for each must be unique.
-    - The "authz_policy" section describes the policy imposed on organizations and roles.  Roles of each organization have specific rights.  Sites of each
-    organization have specific rules.  Rights grant roles to do certain operations.  Rules allow certain operations performed in sites.
-
-.. attention:: Please make sure that the FL server port number is accessible by all participating sides.
-
-
-.. _project_yml:
-
-Default project.yml file
-========================
-
-The following is an example of the default project.yml file.
-
-.. literalinclude:: ../../nvflare/lighter/project.yml
-  :language: yaml
+    - "participants" describes the different parties in the FL system, distinguished by type. For all participants, "name"
+      should be unique, and "org" should be defined in AuthPolicyBuilder. The "name" of the Overseer and servers should
+      be in the format of fully qualified domain names. It is possible to use a unique hostname rather than FQDN, with
+      the IP mapped to the hostname by having it added to ``/etc/hosts``.
```

Comment:
We should be explicit here that when using hostnames and mapping IPs via /etc/hosts, this mapping must be made on each participant system.